### ISSUE TITLE
[Minigo] Pre-trained checkpoint conversion script across frameworks

### ISSUE THREAD

**Issue by nv-rborkar** [mlcommons/training_policies#420]
As per #317 , we would like all references that use pre-trained weights or start from a checkpoint to provide conversion scripts for use across different frameworks (Pytorch, TF, & other frameworks of choice for submission) to ensure submitters use the same pre-approved checkpoint.

AI Intel (reference owner) for Minigo
Example script & recipe in #391

**Comment by johntran-nv** [comment]
@christ1ne  could you assign to the reference owner?

**Comment by christ1ne** [comment]
@johntran-nv are you referring to the following example for converting the weights? https://github.com/mlcommons/training/pull/436
That's a from PyTorch model. Is there an TF model weights export example?
Thanks!

@delock  who's the reference owner.

**Comment by johntran-nv** [comment]
Yes that's what we're referring to.  Sorry but I don't know of a similar TF export example.  We could ask the Google folks for help, if you want.

**Comment by christ1ne** [comment]
I think we should ask the Google folks for an TF example. 
Meanwhile, I am not sure if all the TF models can be converted to Pytorch as they are today.

**Comment by ShriyaRishab** [comment]
Minigo benchmark is retired from the suite, closing.
